src/oic/utils/sdb.py

```diff
@@ -510,7 +510,7 @@ def _get_token_type(self, item, order=None):
         for key in order:
             try:
                 return self.token_factory[key].get_type(item)
-            except Exception:
+            except Exception:  # nosec
```

Comment:
We should really get to these at some point and fix them to be more specific. Could you please add a FIXME, so we do not forget?

Reply:
ok.